### PR TITLE
feat: Add header section for careers page

### PR DESCRIPTION
**Pull request by usame-algan**
## What it solves

Adds a header section to the careers page

## Screenshot
Screenshot 2022-04-27 at 12 50 29

### CODE REVIEW

File: src/components/careers/Header/index.tsx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+const Header = ({ openPositions }: { openPositions: number | undefined }) => {
+  return (
+    <Container>
+      <SLinesSVG />
+      <Wrapper>
+        <ContentWrapper>
+          <div>
+            <Grid item xs={12}>
+              <Text>{openPositions} Positions</Text>
```
Comment (iamacook) [inline src/components/careers/Header/index.tsx:99]: This will look odd if there are no open positions.

**Comment by iamacook** [comment]
The flow is a bit strange with `# Positions` above `Join us`. I would move it below the header.

**Comment by usame-algan** [comment]
> The flow is a bit strange with `# Positions` above `Join us`. I would move it below the header. Maybe you could even move the number onto the button.

I followed the Design but like the idea of displaying the number of open positions within the button instead of above the title.

**Comment by katspaugh** [comment]
So the header from other pages isn't reusable after all?

**Comment by usame-algan** [comment]
> So the header from other pages isn't reusable after all?

Unfortunately, there are no "central" styled components that we can reuse. This can definitely be refactored together with adjustments for typography and spacings but I would suggest to do it in a separate task.

**Pull request merged**